Lib/test/test_peg_parser.py

```diff
@@ -631,6 +631,12 @@ def f():
     ),
 }
 
+EXPRESSIONS_TEST_CASES = [
+    ("expression_add", "1+1"),
+    ("expression_add_2", "a+b"),
+    ("expression_call", "f(a, b=2, **kw)"),
```

Comment:
I see nothing with commas -- is that okay?

Reply:
Good point!